doc/source/release-notes/Percona-Server-5.5.50-38.0.rst

```diff
@@ -0,0 +1,43 @@
+.. rn:: 5.5.50-38.0
+
+==============================
+ |Percona Server| 5.5.50-38.0
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.50-38.0 on July 8th, 2016. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.50-38.0/>`_ and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.50 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-50.html>`_, including all the bug fixes in it, |Percona Server| 5.5.50-38.0 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.50-38.0 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.50-38.0>`_. 
+
+New Features
+============
+
+ |Percona Server| has implemented protocol support for :ref:`TLS 1.1 and TLS 1.2 <extended_tls_support>`. This implementation automatically disables TLS v1.0 support.
```

Comment:
`This implementation automatically disables TLS v1.0 support` sounds too strong. I would say `turns this protocol off by default`.